src/collection.js

```diff
@@ -956,6 +956,8 @@ export default class Collection {
       since: options.lastModified ? `${options.lastModified}` : undefined,
       headers: options.headers,
       retry: options.retry,
+      // Fetch every page by default (see #277 about limiting)
```

Comment:
Is the reference to #277 meant to be an acknowledgement of future work? If so, maybe put `FIXME` or something?